### DISCUSSION TITLE
How to specify different database index other than default 0 in hiredis ?

### DISCUSSION THREAD

**Discussion by sujz** [redis/hiredis#1216] [General]
Hello guys,

As we know, each redis instance supports 16 databases by default, and the default database index is 0.

I don't find any hiredis api support this option, both redisConnect and redisConnectWithOptions have no such option like what Go redis client does.

Here is whtat Go redis client does: 
redisdb := redis.NewClient(&redis.Options{
		Addr:     "localhost:6379", // use default Addr
		Password: "",               // no password set
		DB:       0,                // use default DB
	})

So is it possible to specify different database index other than default 0 in hiredis ?

**Comment by nvkhoi112358** [comment]
I think it is answered here https://github.com/redis/hiredis/issues/1120
